RetroBar/Languages/hrvatski.xaml

```diff
@@ -20,16 +20,25 @@
         <s:String>Dno</s:String>
     </x:Array>
     <s:String x:Key="allow_font_smoothing">_Dopusti izglađivanje fonta</s:String>
-    <s:String x:Key="collapse_tray_icons">Skrati ikone područija obavjesti </s:String>
+    <s:String x:Key="collapse_tray_icons">Sakriji neaktivne ikone </s:String>
     <s:String x:Key="customize">Prilagod_i...</s:String>
     <s:String x:Key="show_clock">Prikaži  _sat</s:String>
     <s:String x:Key="show_multi_mon">Prikaži _na više zaslona</s:String>
     <s:String x:Key="show_quick_launch">Prika_ži brzo pokretanje</s:String>
-    <s:String x:Key="select_location">_Select location...</s:String>
+    <s:String x:Key="select_location">_Odaberi lokaciju...</s:String>
     <s:String x:Key="quick_launch_folder">Odaberi datoteku za brzo pokretanje</s:String>
     <s:String x:Key="show_window_previews">Prikaži prikaz prozora(Umanjena sličica)</s:String>
     <s:String x:Key="use_software_rendering">_Koristi software prikazivanje</s:String>
-    <s:String x:Key="ok_dialog">OK</s:String>
+    <s:String x:Key="multiple_displays">Više zaslona</s:String>
+    <s:String x:Key="show_tasks_on">P_rikaži zadatke na:</s:String>
+    <x:Array x:Key="show_tasks_on_values" Type="s:String">
+        <s:String>Sve programske trake</s:String>
+        <s:String>Ista programska traka kao prozor</s:String>
+        <s:String>Ista programska traka kao prozor i primarna programska traka</s:String>
+    </x:Array>
+    <s:String x:Key="version">Verzija {0}</s:String>
+    <s:String x:Key="visit_on_github">Posjeti na GitHubu</s:String>

```

Comment:
Typo on GitHub?

Reply:
No, that is not a typo its how sentences are structured in Croatian, although most people when translating add a "-" in between .